### DISCUSSION TITLE
Close buffer without closing split

### DISCUSSION THREAD

**Discussion by CptPotato** [helix-editor/helix#1902] [Q&A] [answered]
I often use a vertical split when jumping between two documents a lot. But whenever I close a buffer I have to reopen the split.

Is it currently possible to close a buffer but keep the split open?

**Comment by TornaxO7** [comment]
May I ask what you mean? Could you please provide a little example or something similar?

**Reply by CptPotato** [reply, marked as answer]
I think this behavior has changed. I'm pretty sure when I wrote this, `:bc` would always close the split you have focused.

Now it only closes when there are no other buffers to show, which is fine for my workflow. Showing a new scratch buffer would be a nice option if splits get more configurable (so you don't loose the layout you've set up).
